test/benchmark/parameter_benchmarks/Snakefile

```diff
@@ -2,38 +2,69 @@ configfile: "Repos/iGenVar/test/benchmark/config/caller_comparison_config.yaml"
 
 rule all:
     input:
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="min_var_length"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="max_var_length"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="max_tol_inserted_length"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="max_tol_deleted_length"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="max_overlap"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="partition_max_distance"),
-        expand("results/parameter_benchmarks/plots/{parameter_name}.results.all.png",
-               parameter_name="hierarchical_clustering_cutoff")
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="min_var_length"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="max_var_length"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="max_tol_inserted_length"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="max_tol_deleted_length"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="max_overlap"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
+                parameter_name="partition_max_distance"),
+        expand("results/parameter_benchmarks/{dataset}/plots/{parameter_name}.results.all.png",
+                dataset=["Illumina_Paired_End", "Illumina_Mate_Pair", "MtSinai_PacBio", "PacBio_CCS", "10X_Genomics"],
```

Comment:
the dataset is also always the same

Reply:
Good point, this appears in every benchmark workflow. So I think I will try to make it shorter in another PR.